app/client/ui/DocTour.ts

```diff
@@ -69,8 +92,8 @@ async function makeDocTour(docData: DocData, docComm: DocComm): Promise<IOnBoard
 
     if (validLinkUrl && linkText) {
       body = dom(
-        'div',
-        dom('p', body),
+        'span',
```

Comment:
For plain-text, this used to be `<div><p>body</p>...</div>`, but now is `<span>body...</span>`. Is this change intentional?